src/components/ErrorBoundary.js

```diff
@@ -15,6 +18,27 @@ class ErrorBoundary extends Component {
 
 	static contextType = StoreContext
 
+	refreshFights = () => {
+		const {reportStore} = this.context
+		reportStore.refreshReport()
+	}
+
+	renderRefreshButton = (error) => {
+		if (error instanceof NotFoundError) {
+			return <Header>
+				<div className={styles.refreshheader + ' pull-right'}>
```

Comment:
nit; `classNames(styles.refreshHeader, 'pull-right')` (will need an import)

while it realistically doesn't do much more than join the classes in this use-case, it's "safer" to leave that logic to something else.